server/issue_test.go

```diff
@@ -166,3 +166,55 @@ func TestCheckCharacterSetCollationOwner(t *testing.T) {
 		}
 	}
 }
+
+func TestGetTaskAndDagListByVersion(t *testing.T) {
+	databaseID1 := 1
+	databaseID2 := 2
+	tests := []struct {
```

Comment:
ideally should have 2 test cases:

case 1: just a single database with multiple tasks
case 2: 2 databases, each with multiple tasks

Reply:
Done.